### ISSUE TITLE
Add a visual indicator that a test is running

### ISSUE THREAD

**Issue by remi2j** [jest-community/vscode-jest#420]
Let's say I have a test that failed, then I edit it and save the file. The small icon next to the `test` keyword will remain red for a few seconds, then turn green. This is annoying because while the test is running, I have no way of knowing if the green or red dot is a remain of my previous test, or if the new version of that test still fails.

I think each test should have 3 possible states with a matching UI: error (red dot), success (green dot), and running (probably a spinner).

I noticed the PR #63 is supposed to do something similar, but I can't see it in my editor.

**Comment by orta** [comment]
This is a clever idea - yeah, I would enjoy seeing this

**Comment by simkessy** [comment]
I just started with Jest today but I can't tell when my tests are running, the terminal just updates sporadically as I type. Is this normal?

**Comment by henryjw** [comment]
I agree. I think all the tests that need to be re-executed should display a visual indicator to let the user know that the tests are being executed.

**Comment by jmarceli** [comment]
I believe that this is already implemented see #513 (Running indicator is displayed in the StatusBar).
Maybe for improved visibility running icon should also replace success/error icons?
As for me, it would be much more readable.

What do you think about such enhancement? @orta  @connectdotz

**Comment by mikecann** [comment]
I had a stab at implementing this but it the source wasnt as simple as I was expecting (theres no simple "isLoading" state like there is for TestResult)  so had to abandon it. Would love to see it implemented tho if someone has a weekend to spare.
